### ISSUE TITLE
`wellknown` seems to be removed from CRAN

### ISSUE THREAD

**Issue by darribas** [GDSL-UL/wma#2]
The pakcage `wellknown`, which we use on the Data Architecture lab seems to be removed from CRAN?

https://cran.r-project.org/web/packages/wellknown/index.html

We should try to remove it from our course and replace the functionality with other currently alive libraries.

I think this is the only part where we use it:

https://github.com/GDSL-UL/wma/blob/0b22980688c9a5e8c0984816515d67ded3b8f168/dataarch.qmd#L160-L169

I _think_ this functionality is now available in `sf`:

```r
# Load the WKT representation of the point and convert the WKT representation into WKB format
wkt <- "POINT (-2.977367 53.40753)" %>%
               st_as_sfc %>%
               st_as_binary
# Print the WKB data
wkt
```

Is this OK with you @pietrostefani? If so, I'm happy to update it

**Comment by pietrostefani** [comment]
Sounds good to me @darribas - thanks!

**Comment by darribas** [comment]
Addressed in 7ecbc3eeccdaa0e1e131ec0a2efdd5122fad010c
